src/scss/components/_icons.scss

```
@@ -9,10 +9,23 @@
   font-display: swap;
 }
 
+.circle-icon {
+  background: $primary;
+  padding: 20px;
+  border-radius: 100%;
+}
+
+.label-icon {
+  color: $black !important;
+  vertical-align: bottom;
+}
+
 [class^='ri-'],
 [class*='ri-'] {
   font-family: remixicon, sans-serif !important;
   font-style: normal;
+  font-weight: 100;
+  color: $secondary;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
```

```suggestion
[class*='ri-'] {
  font-family: remixicon, sans-serif !important;
  font-style: normal;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}
```
Les icônes ne devraient pas imposer une couleur de base, on ne sait pas à l'avance dans quelles couleurs elles seront affichées.
Pourquoi changer la `font-weight` ?